scene/gui/popup_menu.cpp

```diff
@@ -1107,9 +1120,13 @@ void PopupMenu::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("_set_items"),&PopupMenu::_set_items);
 	ObjectTypeDB::bind_method(_MD("_get_items"),&PopupMenu::_get_items);
 
+	ObjectTypeDB::bind_method(_MD("set_hide_on_item_selection","enable"),&PopupMenu::set_hide_on_item_selection);
+	ObjectTypeDB::bind_method(_MD("get_hide_on_item_selection"),&PopupMenu::get_hide_on_item_selection);
+
 	ObjectTypeDB::bind_method(_MD("_submenu_timeout"),&PopupMenu::_submenu_timeout);
 
 	ADD_PROPERTY( PropertyInfo(Variant::ARRAY,"items",PROPERTY_HINT_NONE,"",PROPERTY_USAGE_NOEDITOR), _SCS("_set_items"),_SCS("_get_items") );
+	ADD_PROPERTY( PropertyInfo(Variant::BOOL, "hide_on_item_selection" ), _SCS("set_hide_on_item_selection"), _SCS("get_hide_on_item_selection") );
```

Comment:
The property defaults to `true`, so save some memory here by using `ADD_PROPERTYNO`, which will only save the value if it's not `true` (i.e. if it's not 1, `NO` = non-one)